config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -227,7 +227,7 @@ periodics:
           # this is mostly for building kubernetes
           memory: "9000Mi"
           # during the tests more like 3-20m is used
-          cpu: 7400m
+          cpu: 7300m
```

Comment:
any chance of a comment explaining how we arrived here?

Reply:
this approximately is the remaining CPU on the node after the sidecar started using 100m and the GKE overhead.
visibility into this is poor. this was not meant to be a lasting plan.